rpc re-organize and limits added (#1241)
near/docs: docs/5.api/rpc/introduction.md

---
id: introduction
sidebar_label: Home
title: NEAR RPC API
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> The RPC API allows you to communicate directly with the NEAR network. This is how tools such as [near-api-js](/tools/near-api-js/quick-reference)
> actually work, they are just abstractions of making RPC calls.

Reviewer: ```suggestion
The RPC API allows you to communicate directly with the NEAR network. For example,
tools such as [near-api-js](/tools/near-api-js/quick-reference) are just abstractions making RPC calls.
```

<hr class="subsection" />

## RPC Providers

> There are multiple [RPC providers from which you can choose from](./providers.md). These providers will work as intermediaries to help you interact with the NEAR network.

Reviewer: ```suggestion
There are multiple [RPC providers which you can choose from](./providers.md). These providers will work as intermediaries to help you interact with the NEAR network.
```
